src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  justify-content: center;
+  padding-top: 2px;
+}
```

Comment:
You must have a block class such as "stars" to use a modifier, and if you will have it you can apply these properties to it and don't repeat them every time.

Reply:
Problem is fixed